mantle/platform/api/gcloud/image.go

```diff
@@ -102,12 +127,15 @@ func (a *API) CreateImage(spec *ImageSpec, overwrite bool) (*compute.Operation,
 	return op, a.NewPending(op.Name, doable), nil
 }
 
-func (a *API) ListImages(ctx context.Context, prefix string) ([]*compute.Image, error) {
+func (a *API) ListImages(ctx context.Context, prefix string, family string) ([]*compute.Image, error) {
```

Comment:
nit: `ListImages(ctx context.Context, prefix, family string)`

It feels a little awkward to have a filter function input that just as often is `""`, _but_ it works and I can't think of a better way to do it without adding `ListImagesInFamily` or something like that.

Reply:
yeah, kind of icky but I didnt' commit too much time to figuring it out. I guess another option is that we add the caller the ability to specify their own filter, which should be generic enough to cover other use cases

Reply:
Na this is fine. It works. That's enough. We can refine it over time as needed :smile: